learn-arai/app/components/icons/facebook.tsx

```diff
@@ -0,0 +1,14 @@
+import Image from "next/image";
+
+export const FacebookIcon = () => {
```

Comment:
Is there any reasons to use arrow function instead of regular function?
If there is only one export, consider default export this function